### ISSUE TITLE
Design decision 12 - Double H1 on a single page

### ISSUE THREAD

**Issue by SimonaZoc** [wet-boew/wet-boew#9104]
Hi,
I'm looking at this page: https://wet-boew.github.io/wet-boew-documentation/decision/12.html under Guidance.
I was wondering if someone could explain to me the difference between the two below:

These scenarios do not represent WCAG failures:
•Skipped heading levels

These scenarios do represent WCAG failures:
•Illogical heading order

To me a skipped heading is <h1>, <h3>, <h4>
and an illogical heading order is the same thing <h1>, <h3>, <h4>, <h2> and they should be both categorized as a failure.

Example: if  <h2> is missing it can cause confusion to where <h3>, <h4> belong too. 

Thank you in advance!

**Comment by EricDunsworth** [comment]
@SimonaZoc FYI I fixed the heading tags in the OP since GitHub was trying to render them as real HTML elements :).

**Comment by EricDunsworth** [comment]
As for the issue itself, @GormFrank might want to chime in since he wrote that design decision.

**PS:**
FWIW I'm personally not a fan of having multiple H1 headings within a single page.

**Comment by duboisp** [comment]
The use on double `h1` is for a specific use case when there is a set of pages navigation section along with a content section, like when it used by the gc-subway navigation pattern.

Using double `h1` don't means to skip `h2`.

**Comment by GormFrank** [comment]
@SimonaZoc I understand both terms could be interpreted the same. You got the Skipped headings right! For the Illogical heading order, the meaning is more contextual, in the sense that something like this would fail:

```
<h3>Additive primary colors</h3>
<h2>Red</h2>
<h2>Green</h2>
<h2>Blue</h2>
```

Does that make sense? Cheers!
